app/views/early_years_payment/provider/start/claims/_ineligibility_email_not_on_whitelist.html.erb

```diff
@@ -0,0 +1,13 @@
+<% content_for(:page_title, page_title("You do not have access to this service", journey: current_journey_routing_name)) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">You do not have access to this service</h1>
+    <p class="govuk-body">
+      This email address does not have access to this service. Check the email address you entered and try again.
+    </p>
+    <p class="govuk-body">
+      To request access, contact us at <a href="mailto:ops@opsteam.gov.uk">ops@opsteam.gov.uk</a>.
```

Comment:
```suggestion
      To request access, contact us at <%= govuk_mail_to t("early_years_payment_provider_start.early_years_payment_provider_start.support_email_address") %>.
```
`support_email_address` doesn't exist currently for early_years_payment_provider_start - is it supposed to be the same as the `feedback_email`?